File: tests/fixtures/kotlin_hello/Greeter.kt
package app

// Same-file Calls edge: greet() calls format(). Cross-file: format() touches
// the Registry object defined in Registry.kt (same package).
class Greeter(private val name: String) {
    fun greet(): String {
        return format()
    }

    private fun format(): String {
        return Registry.prefix() + name
    }
}

File: tests/fixtures/php_hello/src/Greeter.php
<?php
namespace App;

class Greeter
{
    // Same-file Calls edge: greet() calls format().
    public function greet(string $name): string
    {
        return $this->format($name);
    }

    private function format(string $name): string
    {
        // Cross-file call: format() calls the free function salutation()
        // defined in helpers.php (same namespace).
        return salutation() . ", " . $name;
    }
}
